Pop-Grupa-A-Frontend/src/services/clusterService.js

```diff
@@ -13,6 +13,18 @@ function getClustersForUser(userId){
     return clusterlist;
 }
 
+function submitClusterNode(nodeId){
+    var submitResponse = fetch(`${clusterUrl+nodeId+"/submit"}`, {
+        crossDomain:  true,
+        method: 'post',
+        headers: { 'Content-Type': 'application/json' },
+    })
+    .then(response => response.json())
+    .then(response => {
+        return response;
+    });
```

Comment:
Można to zrobić chyba na jednym .then, wydaje mi się, że drugi nie jest wymagany

Reply:
wydaje mi się że jest to potrzebne bo wywołanie json() też zwróci promise bo odczyt jest asynchroniczny